ts/IterableInboxMessageDisplay.tsx

```diff
@@ -72,6 +73,9 @@ const IterableInboxMessageDisplay = ({
 
    const handleHTMLMessage = (event: any) => {
       let url = event.nativeEvent.data
+
+      Iterable.trackInAppClick(rowViewModel.inAppMessage, IterableInAppLocation.inApp, url)
```

Comment:
for inApps inside inbox, we should send InAppLocation as `Inbox`

Android SDK reference: https://github.com/Iterable/iterable-android-sdk/blob/1784916969d50475377290a6078b44b73c1c2591/iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppLocation.java#L10

Reply:
Yes. I will make the change.